package.json

```diff
@@ -1,27 +1,43 @@
 {
   "name": "frint.js.org",
-  "description": "The FrintJS Website",
+  "description": "The FrintJs Website",
   "private": true,
-  "scripts": {
-    "alex": "alex ."
```

Comment:
It's better to keep `alex`, it allows us to check whether the content is neutral and not offending to anyone: https://www.npmjs.com/package/alex

Reply:
Thanks for the explanation. I added the script and package.